### PR TITLE
Add magichome document

### PR DESCRIPTION
**Pull request by ZNEGGE-SDK**
## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:https://github.com/home-assistant/core/pull/32599
- Link to parent pull request in the brands repository: https://github.com/home-assistant/brands/pull/47
- This PR fixes or closes issue:

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

**Comment by ZNEGGE-SDK** [comment]
https://github.com/home-assistant/home-assistant.io/pull/11110

File: source/_integrations/magichome.markdown [hunk 1]
```diff
@@ -0,0 +1,57 @@
+---
+title: "MagicHome"
+description: "Instructions on how to setup the MagicHome hub within Home Assistant."
+logo: magichome.png
```
Comment (frenck) [inline source/_integrations/magichome.markdown]: Please open a PR to add the brand logo's on our brands repository:

<https://github.com/home-assistant/brands>

Remove the `logo` property after that and link the PR @ the Brands repository to this PR and the parent PR.

Thanks! 👍

File: source/_integrations/magichome.markdown [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+ha_category:
+  - Light
+  - Switch
+ha_iot_class: Cloud Polling
+ha_release: 0.103
```
Comment (frenck) [inline source/_integrations/magichome.markdown]: `ha_domain` is missing.

**Review by frenck — CHANGES_REQUESTED** [review]
Hi there, @ZNEGGE-SDK!

Thanks for the PR. I've reviewed it, and left 2 comments. Could you please take a look? Thanks!

👍

File: source/_integrations/magichome.markdown (hunk 1 above)
Comment (LIULi-VVET) [reply source/_integrations/magichome.markdown]: OK！

File: source/_integrations/magichome.markdown (hunk 2 above)
Comment (LIULi-VVET) [reply source/_integrations/magichome.markdown]: Is there any documentation describing ha_domain?

Comment (frenck) [reply source/_integrations/magichome.markdown]: `ha_domain` must have the same value as the `domain` in the manifest.json file of the integration. A PR to add details for `ha_domain` to our developer documentation, is pending.

Comment (LIULi-VVET) [reply source/_integrations/magichome.markdown]: Thanks!

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Comment by klaasnicolaas** [comment]
You can remove the logo file from this pull request in favor of the brand PR 😉

**Comment by LIULi-VVET** [comment]
> You can remove the logo file from this pull request in favor of the brand PR 😉

OK!

**Comment by frenck** [comment]
Closing PR as the parent PR has been closed.

**Pull request closed without merging**